server/apps/distributed-app/helm-chart/james/templates/deployment.yaml

```diff
@@ -0,0 +1,425 @@
+{{- $elasticsearchUrl := include "elasticsearch.url.list" . }}
+{{- $cassandraUrl := include "cassandra.url.list" . }}
+{{- $jamesOpts := include "james.jvmOpts" . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: james-jmap
```

Comment:
Open question: Shouldn't we go back to something simpler here for the community? Only one James server deployment? No jmap / imap-smtp separate instances?

Reply:
I would say keep it this way. Already cut down all the ldap, lsc and op. Just a config to toggle on/off :P

This is better IMO. Help emphasized that our chart capable of support deploying James in both JMAP/ IMAP mode, distributed, separated workload.

Reply:
Makes it more complex though... People might get lost when trying to change things here

Reply:
Both OK to me